plugins/depthCamera/src/DepthCamera.cc

```diff
@@ -52,12 +52,18 @@ void GazeboYarpDepthCamera::Load(sensors::SensorPtr _sensor, sdf::ElementPtr _sd
     _sensor->SetActive(true);
 
     // Add my gazebo device driver to the factory.
+    #ifndef GAZEBO_YARP_PLUGINS_DISABLE_IMPLICIT_NETWORK_WRAPPERS
     ::yarp::dev::Drivers::factory().add(new ::yarp::dev::DriverCreatorOf< ::yarp::dev::GazeboYarpDepthCameraDriver>
                                         ("gazebo_depthCamera", "RGBDSensorWrapper", "GazeboYarpDepthCameraDriver"));
-
+    #else
+    ::yarp::dev::Drivers::factory().add(new ::yarp::dev::DriverCreatorOf< ::yarp::dev::GazeboYarpDepthCameraDriver>
+                                        ("gazebo_depthCamera", "", "GazeboYarpDepthCameraDriver"));
+    #endif
```

Comment:
From what I can read in http://www.yarp.it/latest/classyarp_1_1dev_1_1DriverCreatorOf.html#a0918fdc89443a9e6217bd02af78c0db6, the wrapper argument should only be empty if there is no wrapper device capable of wrapping this YARP device, but I don't think that's the case. Do you have any specific reason to put the default wrapper here and in the rest of the devices to `""`?

Reply:
I left it blank because now you can have different wrappers depending on what are you doing, so there is no default one like before. Also in the new architecture we want to open the wrappers with yarprobotinterface, so I think is confusing to have it there.

Reply:
Ack.